po/it.po

```diff
@@ -824,219 +824,176 @@ msgid "Scoped Threads"
 msgstr "Thread con Scope (Scoped Threads)"
 
 #: src/SUMMARY.md:262
-#, fuzzy
 msgid "Channels"
-msgstr "Canali"
+msgstr "Canali (Channels)"
 
 #: src/SUMMARY.md:263
-#, fuzzy
 msgid "Unbounded Channels"
-msgstr "Canali illimitati"
+msgstr "Canali illimitati (Unbounded Channels)"
 
 #: src/SUMMARY.md:264
-#, fuzzy
 msgid "Bounded Channels"
-msgstr "Canali delimitati"
+msgstr "Canali delimitati (Bounded Channels)"
 
 #: src/SUMMARY.md:265
-#, fuzzy
 msgid "Send and Sync"
-msgstr "Invia e sincronizza"
+msgstr "Send e Sync"
 
 #: src/SUMMARY.md:265
-#, fuzzy
 msgid "Send"
-msgstr "Inviare"
+msgstr "Send (Inviare)"
 
 #: src/SUMMARY.md:265
-#, fuzzy
 msgid "Sync"
-msgstr "Sincronizza"
+msgstr "Sync (Sincronizzare)"
 
 #: src/SUMMARY.md:268
-#, fuzzy
 msgid "Examples"
 msgstr "Esempi"
 
 #: src/SUMMARY.md:269
-#, fuzzy
 msgid "Shared State"
-msgstr "Stato condiviso"
+msgstr "Stato Condiviso"
 
 #: src/SUMMARY.md:270
-#, fuzzy
 msgid "Arc"
-msgstr "Arco"
+msgstr "Arc"
 
 #: src/SUMMARY.md:271
-#, fuzzy
 msgid "Mutex"
 msgstr "Mutex"
 
 #: src/SUMMARY.md:274 src/SUMMARY.md:294
-#, fuzzy
 msgid "Dining Philosophers"
 msgstr "Filosofi a tavola"
 
 #: src/SUMMARY.md:275
-#, fuzzy
 msgid "Multi-threaded Link Checker"
-msgstr "Verifica link multi-thread"
+msgstr "Verificatore di Link a thread multipli"
 
 #: src/SUMMARY.md:277
-#, fuzzy
 msgid "Concurrency: Afternoon"
 msgstr "Giorno 1: Pomeriggio"
 
 #: src/SUMMARY.md:279
-#, fuzzy
 msgid "Async Basics"
-msgstr "Nozioni di base sull'asincronismo"
+msgstr "Nozioni di base su asincronia (Async)"
```

Comment:
Optional nit: Not sure how to translate it better, but "asincronia" sounds pretty bad. Maybe "Async 101"?